### PR TITLE
Bump Quarkus to 2.7.5.Final

### PR DESCRIPTION
**Pull request by gwenneg**

### CODE REVIEW

File: camel-sender-demo/pom.xml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
     <properties>
 
-        <camel-quarkus.version>2.6.0</camel-quarkus.version>
-        <camel-version>3.8.0</camel-version>
+        <camel-quarkus.version>2.7.0</camel-quarkus.version>
+        <camel-version>3.14.1</camel-version>
```
Comment (gwenneg) [inline camel-sender-demo/pom.xml:18]: @pilhuhn I made these changes based on [this file](https://github.com/apache/camel-quarkus/blob/2aa6d4388bad851d7205424ef746adba616e1465/pom.xml#L42-L43) from the camel-quarkus repository.

File: .rhcicd/clowdapp-backend.yaml [hunk 2]
```diff
@@ -151,9 +151,9 @@ objects:
           value: ${OB_TOKEN_USER}
         - name: OB_TOKEN_PASS
           value: ${OB_TOKEN_PASS}
-        - name: OB_MP_REST_URL
+        - name: QUARKUS_REST_CLIENT_OB_URL
           value: ${OB_MP_REST_URL}
-        - name: OB_KC_REST_URL
+        - name: QUARKUS_REST_CLIENT_KC_URL
```
Comment (gwenneg) [inline .rhcicd/clowdapp-backend.yaml:156]: Fixed a typo in the old name here: `OB_KC_REST_URL` should have been `KC_MP_REST_URL`.

File: .rhcicd/clowdapp-engine.yaml [hunk 3]
```diff
@@ -168,9 +168,9 @@ objects:
           value: ${OB_TOKEN_USER}
         - name: OB_TOKEN_PASS
           value: ${OB_TOKEN_PASS}
-        - name: OB_MP_REST_URL
+        - name: QUARKUS_REST_CLIENT_OB_URL
           value: ${OB_MP_REST_URL}
-        - name: OB_KC_REST_URL
+        - name: QUARKUS_REST_CLIENT_KC_URL
```
Comment (gwenneg) [inline .rhcicd/clowdapp-engine.yaml:173]: Fixed a typo in the old name here: `OB_KC_REST_URL` should have been `KC_MP_REST_URL`.

File: engine/src/main/resources/application.properties [hunk 4]
```diff
@@ -66,16 +66,16 @@ quarkus.log.category."com.redhat.cloud.notifications".level=INFO
...
+quarkus.rest-client.rbac-s2s.read-timeout=120000
 
 # IT User service
 # these entries are needed because of a bug in quarkus: https://github.com/quarkusio/quarkus/issues/8384
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/url=${IT_S2S_MP_REST_URL}/v2
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStore=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStorePassword=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
+quarkus.rest-client.com.redhat.cloud.notifications.recipients.itservice.ITUserService.url=${QUARKUS_REST_CLIENT_IT_S2S_URL}/v2
+quarkus.rest-client.com.redhat.cloud.notifications.recipients.itservice.ITUserService.key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
+quarkus.rest-client.com.redhat.cloud.notifications.recipients.itservice.ITUserService.key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
```
Comment (gwenneg) [inline engine/src/main/resources/application.properties:78]: Checking if this is correct...

File: engine/src/main/resources/application.properties [hunk 5]
```diff
@@ -66,16 +66,16 @@ quarkus.log.category."com.redhat.cloud.notifications".level=INFO
...
+quarkus.rest-client.rbac-s2s.read-timeout=120000
 
 # IT User service
 # these entries are needed because of a bug in quarkus: https://github.com/quarkusio/quarkus/issues/8384
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/url=${IT_S2S_MP_REST_URL}/v2
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStore=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStorePassword=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".url=${QUARKUS_REST_CLIENT_IT_S2S_URL}/v2
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
```
Comment (gwenneg) [inline engine/src/main/resources/application.properties:78]: Based on https://quarkus.io/guides/all-config0

**Comment by gwenneg** [comment]
Quarkus is not happy with the IT S2S configuration. I'm looking into it.

**Comment by gwenneg** [comment]
Problem solved by adding the default `FILL_ME` which will be used when the env vars are not available.

File: engine/src/main/resources/application.properties [hunk 6]
```diff
@@ -66,16 +66,16 @@ quarkus.log.category."com.redhat.cloud.notifications".level=INFO
...
+quarkus.rest-client.rbac-s2s.read-timeout=120000
 
 # IT User service
 # these entries are needed because of a bug in quarkus: https://github.com/quarkusio/quarkus/issues/8384
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/url=${IT_S2S_MP_REST_URL}/v2
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStore=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStorePassword=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".url=${QUARKUS_REST_CLIENT_IT_S2S_URL:FILL_ME}/v2
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE:FILL_ME}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD:FILL_ME}
```
Comment (gwenneg) [inline engine/src/main/resources/application.properties:78]: Problem solved by adding the default `FILL_ME` which will be used when the env vars are not available.

File: .rhcicd/clowdapp-backend.yaml (hunk 2 above)
Comment (pilhuhn) [reply .rhcicd/clowdapp-backend.yaml:156]: Why do we do the `QUARKUS_REST_CLIENT_` prefixing? Is that required now?

File: camel-sender-demo/pom.xml (hunk 1 above)
Comment (pilhuhn) [reply camel-sender-demo/pom.xml:18]: That most likely has no impact at all (compatible change).

**Review by pilhuhn — APPROVED** [review]

File: .rhcicd/clowdapp-backend.yaml (hunk 2 above)
Comment (gwenneg) [reply .rhcicd/clowdapp-backend.yaml:156]: As discussed on Slack, yes this is required. Quarkus stopped supporting the old configuration keys since 2.4.x.

**Pull request merged**